app/services/dttp/defer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Dttp
+  class Defer
+    include ServicePattern
+    include Mappable
+
+    class Error < StandardError; end
```

Comment:
Could we make the name a little more context-specific? Even better if we could give it the message when raising